forms-flow-form-export/src/formsflow_export_api/resources/form_process_mapper.py

```diff
@@ -21,7 +21,7 @@
 API = Namespace("Form", description="Form")
 
 
-@API.route("/<string:form_id>/submission/<string:submission_id>/render")
+@API.route("/<string:form_id>/submission/<string:submission_id>/render", methods=["GET"])
```

Comment:
resource name `form_process_mapper` looks like not appropriate.
Also can you change the implementation to accept the x-jwt-token header and to use it, rather than creating the token all the time ?